ecs-logs-collector.sh

```diff
@@ -496,6 +499,43 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    cat /etc/sysconfig/docker > "$info_system"/docker/sysconfig
+    ok
+  else
+    info "/etc/sysconfig/docker not found"
+
+  fi
+}
+
+get_docker_sysconfig_storage() {
+  try "Gathering Docker storage sysconfig"
+
+  if [ -e /etc/sysconfig/docker-storage ]; then
+    cat /etc/sysconfig/docker-storage > "$info_system"/docker/sysconfig-storage
```

Comment:
There's a few things I'd like to see changed here:

- ensure the directory exists prior to writing (in case of reorder or other conditional steps)
- please use `cp`
- nit: I suggest using a "fuller" name here

```suggestion
    mkdir -p "${info_system}/docker"
    cp /etc/sysconfig/docker-storage "$info_system"/docker/sysconfig-docker-storage
```

Reply:
done